### PR TITLE
feat: CRC20 token whitelist support

### PR DESCRIPTION
**Pull request by XinyuCRO**
Currently only support CRC20 token list on Cronos main net, on other chains(eg. test net), there is no whitelist data, so we consider all tokens are whitelisted and no warnings have to be shown.

- UI
image
image
image

### CODE REVIEW

**Comment by crypto-matto** [comment]
Overall is working great~ There are some suggestions:
1. I would suggest adding a warning message in Asset Detail as well. We may use the same message. 
2. Shall we consider merging both `CRC20_TOKEN_ICON_URL` & `CRC20_MAINNET_WHITELIST_TOKENS` into one `Map`? We may consult @cdc-Hitesh on where to add the Icon Url.

**Review by crypto-matto — DISMISSED** [review]

File: src/config/CRC20Tokens.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
+export interface CRC20TokenInfo {
+  address: string;
+  // is tradable on VVS
+  isWhitelist: boolean;
+  iconURL: string;
+}
+
+// https://cronos.crypto.org/explorer/tokens
+// don't know why all response from tokenlist returns lowercased contract address
+// it's definitely not correct to use lowercased contract address here
```
Comment (cdc-Hitesh) [inline src/config/CRC20Tokens.ts:10]: Can we use checksum addresses?

File: src/models/UserAsset.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ export interface UserAsset {
   // CRC20/ERC20 assets should have a contract address persisted along side them.
   // This will be used later for all actions interacting with the token like tokens transfers, etc, ...
   contractAddress?: string;
+
+  isWhitelisted?: boolean;
```
Comment (cdc-Hitesh) [inline src/models/UserAsset.ts:73]: Is it a good idea to maintain `isWhitelisted` flag  on the `UserAsset` level or maintain a list of whitelisted addresses?
Just Curious to know.
@XinyuCRO

File: src/config/CRC20Tokens.ts [hunk 3]
```diff
@@ -0,0 +1,174 @@
+export interface CRC20TokenInfo {
+  address: string;
+  // is tradable on VVS
+  isWhitelist: boolean;
```
Comment (cdc-Hitesh) [inline src/config/CRC20Tokens.ts:4]:
```suggestion
  isWhitelisted: boolean;
```

**Review by cdc-Hitesh — COMMENTED** [review]
PR is strong.
Left a few comments.

File: src/models/UserAsset.ts (hunk 2 above)
Comment (XinyuCRO) [reply src/models/UserAsset.ts:73]: Yeah, I believe it just likes `iconURL` and other properties, are Asset's attributes, and it's easy to use from outside

File: src/config/CRC20Tokens.ts (hunk 1 above)
Comment (XinyuCRO) [reply src/config/CRC20Tokens.ts:10]: Good point!

**Review by cdc-Hitesh — APPROVED** [review]

**Review by crypto-matto — APPROVED** [review]
LGTM Good work

**Pull request merged**